selfdrive/car/hyundai/values.py

```diff
@@ -1369,16 +1363,6 @@ class Buttons:
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00NX4 FR_CMR AT USA LHD 1.00 1.00 99211-N9240 14Q',
     ],
-    (Ecu.eps, 0x7d4, None): [
```

Comment:
do we have a test that prevents these from being included if they're not in the whitelist?

Reply:
We have no whitelist for specific cars. We'd need to check CANFD_CAR

Reply:
made a hyundai specific test file for this